controllers/argocd/configmap_test.go

```diff
@@ -31,6 +31,7 @@ import (
 
 	"github.com/argoproj-labs/argocd-operator/api/v1alpha1"
 	argoprojv1alpha1 "github.com/argoproj-labs/argocd-operator/api/v1alpha1"
+	argov1alpha1 "github.com/argoproj-labs/argocd-operator/api/v1alpha1"
```

Comment:
This is a duplicate import. 
`argoprojv1alpha1` and `argov1alpha1` are both the same.